flow/embedded_views.h

```diff
@@ -365,6 +365,13 @@ class DisplayListEmbedderViewSlice : public EmbedderViewSlice {
 //
 // Used on iOS, Android (hybrid composite mode), and on embedded platforms
 // that provide a system compositor as part of the project arguments.
+//
+// The lifecycle of drawing a frame using ExternalViewEmbedder is:
+//
+//   1. At the start of a frame, call |BeginFrame|, then |SetUsedThisFrame| to
+//      true.
+//   2. For each view to be drawn, call |PrepareView|, then |SubmitView|.
```

Comment:
View versus platform view is confusing. Would it be better to refer to views that back Flutter content as "Flutter views"?

Reply:
The status is:
* There are native views, each one corresponding to a window (simply put).
* The content for each native view is composited from a layer tree.
* Layer trees have types, platform view layer being one of them. 
  * I'm not sure if a "flutter view" is the complement for "platform view".

Reply:
I would be tempted to use the term Flutter View and Platform View explicitly where there's any possibility for confusion.

Reply:
I changed the names to `PrepareFlutterView` `SubmitFlutterView` `flutter_view_id` and `platform_view_id`.